requirements.txt

```diff
@@ -0,0 +1,57 @@
+-e git+https://github.com/bitprophet/alabaster/@3b68afcfe55a80508254b22904294100a160e6a7#egg=alabaster
```

Comment:
I think this file should be cleaned up leaving only sphinx- and markdown-related dependencies. Othervise they can raise security alerts